packages/opencensus-core/src/trace/sampler/sampler.ts

```diff
@@ -15,61 +15,45 @@
  */
 
 import {randomSpanId} from '../../internal/util';
-import * as types from './types';
+import {Sampler} from './types';
 
 
 const MIN_NUMBER = Number.MIN_VALUE;
 const MAX_NUMBER = Number.MAX_VALUE;
 
 
-/** This class represent the probability of a tracer. */
-export class Sampler implements types.Sampler {
-  private idUpperBound: number;
-  description: string;
-
-  /**
-   * Constructs a new Sampler instance.
-   */
-  constructor() {}
+/**  Sampler that samples every trace. */
+export class AlwaysSampler implements Sampler {
+  description = 'always';
```

Comment:
Do we still need the `description` field? Now if you want to figure out what a sampler is doing or the probability, you can use
```
if (sampler instanceof ProbabilitySampler) {
  let prob = sampler.getProbability();
}
```

Reply:
> Do we still need the description field? 

Maybe we don't, but Java implementation has a description also.  I would like to keep it if you don't mind.

Reply:
Ok. Let's make it and the corresponding interface field `readonly`. Also, please add code documentation on its purpose in the interface.